Teapot.podspec

```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do | spec |
+  spec.name = "Teapot"
+  spec.version = "2.0.0"
+  spec.summary = "A light-weight URLSession wrapper for building simple API clients"
+  spec.homepage = "https://github.com/elland/Teapot"
+  spec.license = { type: 'MIT' }
+  spec.authors = { "Igor Ranieri": "igor@elland.me" }
+
+  spec.ios.deployment_target = '10.0'
```

Comment:
Should probably also add deployment targets for other supported platforms (watch, tv, macOS)

Reply:
That might be more work than I have time for at the moment. :/

Reply:
Fair enough - since this only specifies iOS deployment target it should ignore the other ones when installing via cocoapods